nmrfx-core/src/main/java/org/nmrfx/chemistry/relax/OrderPar.java

```diff
@@ -193,6 +187,7 @@ public OrderPar set(String name, Double val, Double err) {
                 Ss2, Ss2err,
                 sumSqErr, nValues, nPars, model);
         switch (name) {
+            case "Order_param":
             case "S2":
                 newPar.value = val;
                 newPar.error = err;
```

Comment:
If "Order_param" and "S2" are supposed to be doing the same thing I think you can write it like this:
```suggestion
            case "Order_param", "S2":
                newPar.value = val;
                newPar.error = err;
```